app/data/get-data.js

```diff
@@ -5,9 +5,9 @@ const { getEventsByCorrelationId } = require('./events')
 const getData = async (category, value) => {
   switch (category) {
     case FRN:
-      return await getEventsByFrn(value)
+      return getEventsByFrn(value)
     case CORRELATION_ID:
-      return await getEventsByCorrelationId(value)
+      return getEventsByCorrelationId(value)
```

Comment:
Why were the awaits removed here?

Reply:
`return await` should only be used in a try catch block.

We want to return the promise, not the result of the promise.  The calling function is already waiting.